win32ss/user/ntuser/focus.c

```diff
@@ -1051,6 +1069,9 @@ co_IntSetActiveWindow(
       return FALSE;
    }
 
+   if (Wnd->ExStyle & WS_EX_NOACTIVATE)
```

Comment:
Redundant with the test at line 1092

Reply:
Deleted duplicate code.

Reply:
Actually now we have a bug, because this `Wnd->ExStyle` dereference is done before validating and checking the Wnd pointer (that's why the code at line 1092 was OK but not that one at line 1072).

Reply:
I double-checked: @katahiromz properly fixed @HBelusca s concern before he merged this PR ultimately.

Reply:
Thanks for double-checking :+1: